src/imminent/result.clj

```diff
@@ -113,3 +97,54 @@
     (if (= imminent.result.Failure k)
       @this
       not-found)))
+
+;;
+;; Applicative support
+;;
+
+;; Shared implementation of Applicative for Success and Failure.
+(defn sf-fapply
```

Comment:
Shared `fapply` implementation. (See the PR note on the unified behavior of Success / Failure vs their independence.)